### DISCUSSION TITLE
Error: null is not an object (evaluating 't.parallelRoutes.get')

### DISCUSSION THREAD

**Discussion by JClackett** [vercel/next.js#67729] [Help]
### Summary

Absolutely no idea where this error is coming from, but its deep within got nextjs routing code and only happens sometimes for certain users. 

Only app router stuff, and impossible to replicate, any ideas?

**Comment by just2102** [comment]
https://github.com/vercel/next.js/pull/75360
this has been fixed here

**Reply by just2102** [reply]
still in canary tho
